HotelReservation/src/main/java/HotelList.java

```diff
@@ -0,0 +1,43 @@
+import java.util.ArrayList;
+
+public class HotelList {
+    private ArrayList<Hotel> listOfHotels;
+
+    HotelList(){
+        listOfHotels = new ArrayList<Hotel>();
+    }
+    public void addHotel(Hotel h){
+        listOfHotels.add(h);
+    }
+
+    public void removeHotel(Hotel h){
+        listOfHotels.remove(h);
+    }
+
+    public int size(){
+        return listOfHotels.size();
+    }
+
+    public Boolean contains(String h){
+        for(int i = 0; i < listOfHotels.size(); ++i){
```

Comment:
you don't need to explicitly iterate over the list. use api provided by java https://www.tutorialspoint.com/java/util/arraylist_contains.htm

Reply:
This function just takes a string `h` as hotelname but, the arraylist contains hotel objects which are not comparable with string `h`.